src/sparseml/transformers/sparsification/trainer.py

```diff
@@ -168,30 +167,16 @@ def apply_manager(self, epoch: float, checkpoint: Optional[str]) -> bool:
         :return: True if recipes were applied, False otherwise
         """
 
-        if (not self.arch_managers and self.manager is None) or self.manager_applied:
+        if self.manager is None or self.manager_applied:
             return False
 
         orig_state_dict = self.model.state_dict()
+        self.manager.apply_structure(self.model, epoch=epoch)
```

Comment:
we should also be applying the checkpoint recipe here, not the main recipe manager. what the arch managers were doing here was applying the previous recipes to reproduce graph state.